webpack.config.js

```diff
@@ -0,0 +1,46 @@
+const path = require("path");
+const HtmlWebPackPlugin = require("html-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+module.exports = {
+  devServer: {
+    port: 4000
+  },
+  entry: {
+    "app": "./src/index.js"
+  },
+  output: {
+    path: path.resolve(__dirname, './dist'),
+    filename: '[name].js',
+    chunkFilename: '[id].[chunkhash].js'
+  },
+  devtool: "source-map",
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
```

Comment:
Not sure how difficult it is, and probably outside of the scope of this pull request, but it would be nice to add support for TypeScript. I guess we can create a card for this.

Reply:
That'd be cool. Yeah I say we do that in a new card (off the top of my head we probably will need to setup a tsconfig, install typescript dependencies, update eslint for typescript, and update this line in webpack)